packages/utils/src/get-section-from-schedules.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async ({ content, siteId, load }) => {
+  const { sectionQuery } = content;
+  const query = {
+    status: 1,
+    'site.$id': siteId,
+    alias: { $ne: 'home' },
+  };
+  if (sectionQuery.length) {
+    const sectionsFromScheds = sectionQuery.filter(schedule => schedule.siteId === siteId);
+    const foundSections = await Promise.all(sectionsFromScheds.map(section => load('websiteSection', section.sectionId, {}, query)));
```

Comment:
Immediately bail if no sectionId is found, e.g. `if (!sectionId) return null`

Then do a _single_ load for the sectionId, not a Promise.all of many